tools/RAiDER/utilFcns.py

```diff
@@ -136,27 +149,28 @@ def rio_open(fname, returnProj=False, userNDV=None, band=None):
         # If user requests a band
         if band is not None:
             ndv = nodata[band - 1]
-            data = src.read(band)
+            data = src.read(band).squeeze()
```

Comment:
Is this needed for getting rid of 1-D axes? E.g. [1,20,20] --> [20,20]?

Reply:
yes exactly. and it raises problems down the line without it because it thinks [1,20,20] is a 3D dataset when it's not